nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -147,6 +147,7 @@ public void returnTypeParamInstantiation() {
             "    static class NullableTypeParam<E extends @Nullable Object> {}",
             "    // BUG: Diagnostic contains: Generic type parameter",
             "    static NonNullTypeParam<@Nullable String> testBadNonNull() {",
+            "    // BUG: Diagnostic contains: Cannot assign from type",
```

Comment:
We will want a separate error message type for returns, the equivalent of `RETURN_NULLABLE` but for generics